templates/includes/records/record-details-community.html

```diff
@@ -78,4 +78,25 @@
         <th scope="row">{{ 'type'|as_label }}</th>
         <td>{{ record.type }}</td>
     </tr>
-{% endif %}
\ No newline at end of file
+{% endif %}
+{% if record.provenance %}
+    <tr>
+        <th scope="row">{{ 'provenance'|as_label }}</th>
+        <td>{{ record.provenance }}</td>
+    </tr>
+{% endif %}
+{% if record.place_of_deposit %}
+    <tr>
+        <th scope="row">{{ 'place_of_deposit'|as_label }}</th>
+        <td>
+            {{ record.place_of_deposit.mediator }}
+            {{ record.place_of_deposit.repository }}
```

Comment:
1) Why is this restricted to particular values?
2) The attribute returns `dict` that can contain many values.  Could this be dynamic to accommodate those values?
3) Do we want to display these values at the side of each other, than some unordered list?